flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/avro/AvroParquetReaders.java

```diff
@@ -46,7 +46,8 @@
      */
     public static <T extends SpecificRecordBase> AvroParquetRecordFormat<T> forSpecificRecord(
             final Class<T> typeClass) {
-        return new AvroParquetRecordFormat<>(new AvroTypeInfo<>(typeClass), SpecificData.get());
+        return new AvroParquetRecordFormat<>(
```

Comment:
These changes should be in the previous commit.

Reply:
thanks